Framework/HistogramData/inc/MantidHistogramData/FixedLengthVector.h

```diff
@@ -121,10 +120,6 @@ template <class T> class FixedLengthVector {
    * allow for length modifications, which we need to prevent. */
   std::vector<double> &mutableRawData() { return m_data; }
 
-  // This is used as base class only, cannot delete polymorphically, so
```

Comment:
Was there a warning about this? The comment suggests it was there to avoid being used standalone.

Reply:
This caused a compilation failure since the compiler generated move constructor couldn't destruct the object afterwards

I was a bit hesitant removing it, but it doesn't stop someone taking a ref or a pointer to a FixedLengthVector, only a local so I decided to drop it

Reply:
I see. As long as the type is safe to use standalone then that seems fine.